clients/client-core/src/client/real_messages_control/mod.rs

```diff
@@ -30,6 +30,14 @@ use crate::client::reply_key_storage::ReplyKeyStorage;
 mod acknowledgement_control;
 mod real_traffic_stream;
 
+/// The `RealMessagesController` through the `OutQueueControl` listens to closed connections.
+// WIP(JON) This guy is used across 1) native client, 2) socks5 client, 3) network-requester 4)
+// client-core. Currently the definition is duplicated, and the type sent is `ConnectionId` in the
+// socks5 common code, but u64 elsewhere.
+// Need to figure out a better way to share this.
+pub type ClosedConnectionSender = mpsc::UnboundedSender<u64>;
```

Comment:
That's a tricky one, isn't it. Maybe we need another common crate for client connections?

Reply:
Yep, created `client-connections` crate